.travis.yml

```diff
@@ -68,7 +68,16 @@ install:
     fi
   # Always install from pypi
   - pip install $PRE pep8 cycler
+  # Remove this once we can run all tests in pytest
   - 'pip install https://github.com/tacaswell/nose/zipball/mnt_py36_compat#egg=nose'
+  - pip install pytest
+
+  # some handy plugins for pytest, is this the right place to install them?
+  # https://pypi.python.org/pypi/pytest-cov for docs
+  - pip install pytest-cov
+  # https://pypi.python.org/pypi/pytest-pep8 for docs
+  - pip install pytest-pep8
```

Comment:
This is the right place to install them.  You could put them all on one line:

`pip install pytest pytest-cov pytest-pep8`